### PR TITLE
Upgrading Spring boot to 3.2.5

### PR DESCRIPTION
**Pull request by kth13**
Upgrade to spring-boot 3.2.5:
 - This includes an [upgrade to Spring Boot 3.2.4](https://github.com/spring-projects/spring-boot/releases/tag/v3.2.5)

Version Upgrades:
- org.springframework:spring-web [6.1.5 -> 6.1.6]

### CODE REVIEW

File: release-notes/5.x.md [hunk 1]
```diff
@@ -11,6 +11,14 @@ From the CVE:
 > list. The decoder cumulates bytes in the `undecodedChunk` buffer until it can decode a field, this field can cumulate
 > data without limits. This vulnerability is fixed in 4.1.108.Final.
 
+# 5.15.5
```
Comment (petergphillips) [inline release-notes/5.x.md:14]: should be 5.15.6 and at top of the file

File: release-notes/5.x.md [hunk 2]
```diff
@@ -11,6 +11,14 @@ From the CVE:
...
+
+## Upgrade to spring-boot 3.2.5
+This includes an [upgrade to Spring Boot 3.2.5](https://github.com/spring-projects/spring-boot/releases/tag/v3.2.5)
+
+## Version Upgrades
+- org.springframework:spring-web [6.1.5 -> 6.1.6]
```
Comment (petergphillips) [inline release-notes/5.x.md:20]: don't think this needs to be included as you haven't upgraded spring framework

File: release-notes/5.x.md (hunk 1 above)
Comment (kth13) [reply release-notes/5.x.md:14]: Hi Peter, Yes got it. I didn't notice that. I will change it now

File: release-notes/5.x.md (hunk 2 above)
Comment (petergphillips) [reply release-notes/5.x.md:20]: instead I would include
```
 - org.springframework.boot:spring-boot-gradle-plugin [3.2.4 -> 3.2.5]
 ```

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**